app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -23,6 +24,10 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 			$rootScope.$on('assignmentStateChanged', function (event, data) {
 				scope.view.state = data;
 				scope.renderTable();
+
+				if (scope.showStaffTable == true) {
```

Comment:
Does `renderTable()` render everything else, and then `renderStaffTable()` renders just 'The Staff'? If so, why not simply update `renderTable()` to also handle rendering The Staff? Is there a precedent or reason to separate them?